stripper/ze_dreamin_v3_1.cfg

```diff
@@ -10,4 +10,39 @@ modify:
 	{
 		"min_use_angle" "0.2"
 	}
-}
\ No newline at end of file
+}
+
+;Stop Trolling on Stage 1 and 2 endings
+modify:
+{
+	match:
+	{
+		"classname" "math_counter"
+		"targetname" "blade_hp"
+	}
+	insert:
+	{
+		"OnHitMin" "heal_ef,Kill,4,-1"
+		"OnHitMin" "wind_ef,Kill,4,-1"
+		"OnHitMin" "particlee_snow,Kill,4,-1"
+		"OnHitMin" "freezer_ef,Kill,4,-1"
+		"OnHitMin" "cmd,Command,say *** All Human Item Particle will be disabled in 4 seconds ***,0.1,-1"
```

Comment:
This message (and the one below) aren't really necessary anymore if you're just stopping the particles from displaying.